core/src/main/scala/kafka/coordinator/group/GroupMetadata.scala

```diff
@@ -433,6 +448,17 @@ private[group] class GroupMetadata(val groupId: String, initialState: GroupState
     }
   }
 
+  def maybeInvokeSyncCallback(member: MemberMetadata,
```

Comment:
Can we have some simple unit tests in `GroupMetadataTest` for this function and `replaceGroupInstance`?